src/script/main/app.ts

```diff
@@ -495,16 +498,19 @@ export class App {
         });
       }
 
+      eventLogger.log('setup-mls');
       telemetry.timeStep(AppInitTimingsStep.UPDATED_FROM_NOTIFICATIONS);
       telemetry.addStatistic(AppInitStatisticsValue.NOTIFICATIONS, totalNotifications, 100);
-
       onProgress(97.5, t('initUpdatedFromNotifications', this.config.BRAND_NAME));
 
       const clientEntities = await clientRepository.updateClientsForSelf();
-      await eventTrackerRepository.init(propertiesRepository.getUserConsentStatus().isTelemetryConsentGiven);
+
+      // not waiting for promise to unblock app initialization
```

Comment:
We don't really unblock it. It just appears to the user that the app is done. The user still needs to wait until every message is loaded before being able to see a call, etc. :/

Reply:
Would you like to suggest a better name?

Reply:
We unblock the lock screen by loading this code asynchronously, to make it appear to the user that the app is done loading earlier.